internal/app/render.go

```diff
@@ -41,13 +44,14 @@ func (a *App) render(
 		}
 		return a.renderFile(inputFile, output, data)
 	} else if len(inputDir) > 0 { // Render input directory
-		return a.renderDir(inputDir, outputDir, excluded, data)
+		// if outputDir is empty
```

Comment:
what's this comment?

Reply:
forgot about it sorry, was during dev
deleting